packages/mwp-app-render/src/components/uxcapture/UXCaptureFont.jsx

```diff
@@ -0,0 +1,28 @@
+// @flow
+import React from 'react';
+
+type Props = {
+	fontFamily: string,
+	mark: string,
+};
+
+export const fontLoaderJS = `<script src="https://ajax.googleapis.com/ajax/libs/webfont/1.5.10/webfont.js" />`;
+
+const generateUXCaptureFontJS = (fontFamily: string, mark: string) => `
+    <script>
+        WebFont.load({
+            custom: {
+                families: ['${fontFamily}']
+            },
+            active: function() {
+                window.performance.mark('${mark}');
+            },
+        });
+    </script>
+`;
+const UXCaptureFont = ({ fontFamily, mark }: Props) =>
+	<script
```

Comment:
similar here - `generateUXCaptureFontJS` should probably not specify `<script>` wrapping tag